amlb/runners/docker.py

```diff
@@ -111,17 +119,20 @@ def _upload_image(self, image):
 
     def _generate_script(self, custom_commands):
         docker_content = """FROM ubuntu:18.04
-
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update
-RUN apt-get -y install apt-utils dialog locales
+RUN apt-get update && apt-get install -y tzdata
+RUN apt-get -y install apt-utils dialog locales sudo
```

Comment:
please add `tzdata` here, looks more natural than on the previous line

Reply:
Check if it's better now.